langchain/src/stores/message/redis.ts

```diff
@@ -57,19 +56,7 @@ export class RedisChatMessageHistory extends BaseListChatMessageHistory {
     const orderedMessages = rawStoredMessages
       .reverse()
       .map((message) => JSON.parse(message));
-    const previousMessages = orderedMessages
```

Comment:
Ah so it was being assigned empty values before going to the `mapStoredMessagesToChatMessages` :( thanks for catching